### PR TITLE
Feature/rpcserver lint

### PR DESCRIPTION
**Pull request by ghost**
fixing cpplint errors in rpcserver.

since we now ignore runtime/references lint errors, i did not convert non-const references to pointers.

### CODE REVIEW

File: rpcserver/source/rpcserver.cc [hunk 1]
```diff
@@ -13,12 +15,17 @@ RPCServer::RPCServer(int p) : port{p} {}
...
+    int16_t *version,
+    int16_t *service,
+    int16_t *auth_protocol
+) {
   // handshake header.
   // Handshake has 7 bytes.
-  constexpr size_t handshake_len = 7;
-  char data[handshake_len];
-  asio::error_code error = read_full(sock, asio::buffer(data, handshake_len));
+  constexpr size_t kHandshakeLen = 7;
```
Comment (LINKIWI) [inline rpcserver/source/rpcserver.cc:26]: What is the motivation for this change?

**Review by LINKIWI — APPROVED** [review]

File: rpcserver/source/rpcserver.cc (hunk 1 above)
Comment (ghost) [reply rpcserver/source/rpcserver.cc:26]: there was a lint error saying it should be named like kSomething since it's a constant
